code/obj/item/device/pda2/pda2.dm

```diff
@@ -838,6 +860,30 @@
 		src.underlays += src.ID_image
 		src.updateSelfDialog()
 		user.UpdateName()
+
+	proc/eject_pen(var/mob/user as mob)
+		if (src.pen)
+			if (istype(user))
+				user.put_in_hand_or_drop(src.pen)
+			else
+				var/turf/T = get_turf(src)
+				src.pen.set_loc(T)
```

Comment:
Dumb idea: Would be cool if there was a chance it would shoot off to a nearby turf.

Reply:
i mean, it's funny, but i also like not throwing my pens everywhere 😓